script/convert_import.rb

```diff
@@ -0,0 +1,40 @@
+require "csv"
```

Comment:
No a huge deal, but I'd prefer to see ocassionally used scripts in `bin/`, rather than `script/`, as the latter seems to be more for commonly used scripts to rule them all type things

Reply:
I thought `bin` was for executables that come from gems?

Reply:
I'm pretty sure it can be for any executable, but there's already a Ruby executable in `script/` already, so let's just leave it as is 👍